Python/Setup/swix/Components/CPython2x64.res.swr

```diff
@@ -0,0 +1,6 @@
+use vs
+
+vs.localizedResources
+  vs.localizedResource language=en-us
+                       title="CPython 2.7"
+                       description="Python is a programming language that lets you work more quickly and integrate your systems more effectively. Python 2.7 is a legacy version used in many existing systems."
```

Comment:
One day we will need to localize this file. I'm assuming it'll be easier if it is following the original format rather than some clever generator.

---
In reply to: [69643432](https://github.com/Microsoft/PTVS/pull/1396#discussion_r69643432) [](ancestors = 69643432)